docs: Sustainability effort (Incentive system) (#12359)
argoproj/argo-workflows: community/README.md

Welcome to the Argo community!

> Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done. [This document](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure of the Argo Community including the roles, responsibilities and processes that govern Argo projects and community.

Reviewer: ```suggestion
Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done.
The [Argoproj Community README](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure and governance of Argo projects including roles, responsibilities, and processes.
```

Might as well update this part of the doc while at it too:
- Avoid ambiguous "this document", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
- Prefer 1 sentence per line of markdown [per style guide](https://argoproj.github.io/argo-workflows/doc-changes/)
- Simpler and more direct language, [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

> ## Argo Workflows Sustainability Effort

Reviewer 2: We can also link this in contributing guide

Author: good idea, will do

> Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. Please see [this document](sustainability_effort.md) for more information.

Reviewer: ```suggestion
Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable.
Please see [Sustainability Effort](sustainability_effort.md) for more information.
```

- Avoid ambiguous "this document", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
- Prefer 1 sentence per line of markdown [per style guide](https://argoproj.github.io/argo-workflows/doc-changes/)

Author: Will do. Just curious: I'm not sure if I understand "prefer 1 sentence per line of markdown". I can see that it could sometimes be more readable that way. But do you think it should be followed when there are multiple sentences that flow well together? It does say "prefer", so it's probably not a hard and fast rule.?
